jbpm-wb-case-mgmt/jbpm-wb-case-mgmt-client/src/test/java/org/jbpm/workbench/cm/client/list/CaseInstanceListPresenterTest.java

```diff
@@ -107,6 +107,22 @@ public void testDestroyCaseInstance() {
                      captor.getValue().size());
     }
 
+    @Test
+    public void testCloseCaseInstance() {
+        final CaseInstanceSummary cis = caseInstanceSummaryList.remove(0);
```

Comment:
there is a test already, named _testDestroyCaseInstance_ that should be renamed to close and fixed instead of adding a new test. It is testing the same logic.